lib/protocol/sequences/Sequence.js

```diff
@@ -52,6 +52,7 @@ Sequence.prototype._packetToError = function(packet) {
   err.code = code;
   err.errno = packet.errno;
   err.sqlState = packet.sqlState;
+  err.sql = this.sql;
```

Comment:
And, because `this.sql` is not defined in the constructor of `Sequence`, I think the adding of this property should be done by the object that is handing that property and wrapping this super method.